styleguide/_static/issue-templates/feature.md

```diff
@@ -0,0 +1,47 @@
+---
+name: ✨ Feature
+about: Add/Implement Feature
+title: ✨ <Insert Title>
+labels: feature
+assignees: ''
+
+---
+
+# Summary
+<!---
+A short description of the feature (TL;DR)
+--->
+
+# Details
+<!---
+A detailed description about what should be added, background, links, context information
+and usage example(s) of the feature.
+--->
+
+## Background & Context
+<!---
+What is the motivation for the feature?
+Assessment of the feature. Pros & cons, solutions and decisions concerning the feature (reasoning).
+--->
+
+## Examples
+<!---
+Example usage(s) of the feature.
+--->
+
+## References
+<!---
+Further references to e.g. other information resources like links to specification(s) etc.
+--->
+
+# Task(s)
+<!---
+A task list containing common task associated with this kind of issue, but also tasks specific to this issue.
+
+e.g.:
+- [ ] Refactor XYZ
+- [ ] Bump version number
+- [ ] Update documentation
+- [ ] Update changelog
+- [ ] ...
```

Comment:
same questions as above....keep in issue or move to PR?

Reply:
same answer as above ;)